src/stripe/objects/core/charge.cr

```diff
@@ -78,44 +78,68 @@ class Stripe::Charge
 
   getter id : String
   getter amount : Int32
+  getter amount_captured : Int32?
   getter amount_refunded : Int32
   getter application : String?
   getter application_fee : String?
-  getter balance_transaction : String?
+  getter application_fee_amount : Int32?
+  getter balance_transaction : String? # Expandable to Stripe::BalanceTransaction
+
+  class BilingDetails
```

Comment:
I think this and the references will need to be `BillingDetails`!

Reply:
Of course !